src/main/java/org/testobject/espressorunner/EspressoRunner.java

```diff
@@ -119,34 +116,25 @@ private TestObjectClient createClient() {
 		return request;
 	}
 
-	private void writeSuiteReportXML(TestObjectClient client, String user, String app, long suiteReportId) throws IOException {
+	private void writeSuiteReportXML(TestObjectClientV2 client, String user, String project, long suiteReportId) throws IOException {
```

Comment:
Is this the only thing we actually need the user/project for, if they're using an API key? Can we get rid of this requirement and use something else for the filename instead? Or read the user/project from the API?

Reply:
yes, I agree, just have no idea what to use for filename instead of username. Not sure if we change API to return user/project.